recipes/eastl/all/conanfile.py

```diff
@@ -76,5 +90,7 @@ def package_info(self):
         self.cpp_info.names["cmake_find_package"] = "EASTL"
         self.cpp_info.names["cmake_find_package_multi"] = "EASTL"
         self.cpp_info.libs = ["EASTL"]
-        if self.settings.os in ("Android", "Linux", "Macos", "watchOS", "tvOS"):
+        if self.settings.os == "Linux":
```

Comment:
are you sure about Android? Asking because I have no idea about

Reply:
No, but AFAIK linking pthread on Android is useless.